preprocessors/semanticSeg/segment.py

```diff
@@ -162,6 +170,7 @@ def run_segmentation(url,
     output_size = img_data.shape[1:]
     with torch.no_grad():
         # get segmentation results
+        logging.info("running segmentation model")
```

Comment:
If the below lines may take substantial time (especially in degenerate cases), suggest having logging for each of the steps so we know where it halts and dies. Also, in line 181, there is another loop where the number of iterations should ideally be logged so we can see if anything crazy is happening.